erpnext/vehicles/doctype/vehicle_booking_order/vehicle_booking_order.py

```diff
@@ -1040,3 +1054,68 @@ def update_allocation_booked(vehicle_allocation, is_booked, is_cancelled):
 	is_cancelled = cint(is_cancelled)
 	frappe.db.set_value("Vehicle Allocation", vehicle_allocation, {"is_booked": is_booked, "is_cancelled": is_cancelled},
 		notify=True)
+
+
+@frappe.whitelist()
+def send_customer_vehicle_anniversary_notifications():
+	if not automated_vehicle_anniversary_enabled():
+		return
+
+	now_dt = now_datetime()
+	date_today = getdate(now_dt)
+
+	reminder_dt = get_anniversary_scheduled_time(date_today)
+	if now_dt < reminder_dt:
+		return
+
+	notification_last_sent_date = frappe.db.get_default("vehicle_anniversary_notification_last_sent_date")
+	if notification_last_sent_date and getdate(notification_last_sent_date) == date_today:
```

Comment:
Lets use greater than or equal to operator instead
`getdate(notification_last_sent_date) >= date_today`